### DISCUSSION TITLE
Show metadata in admin web interface

### DISCUSSION THREAD

**Discussion by some1ataplace** [btcpayserver/btcpayserver#3568] [Ideas / Feature Requests]
When programmatically passing metadata to something like the invoice creation using the API, I cannot view the metadata that got passed to the invoice in the btcpayserver admin web interface. 

https://docs.btcpayserver.org/API/Greenfield/v1/#operation/Invoices_CreateInvoice

The only way to see the metadata is to get the invoice:

https://docs.btcpayserver.org/API/Greenfield/v1/#operation/Invoices_GetInvoice

It would be nice to see the metadata in the web interface without needing to programmatically get the invoice via the API each time.

**Comment by Kukks** [comment]
If you want to show the data in the invoice details, create a nested item under `posData`.

So example:
```json
"metadata": {
  "posData": {
		"something": [1,2,3,4,5],
		"somethingElse": "really?",
		"magic": {
			"nah": [ "item", "item2"]
		}
	}
}
```

**Reply by some1ataplace** [reply]
Thank you for providing an example. I also see it now in the docs. Did not realize it was contained in one of the many object buttons.
